README.md

```diff
@@ -177,7 +177,7 @@ Due to their public nature, GitHub and mailing lists are NOT appropriate places
 
 ## etcd Emeritus Maintainers
 
-These emeritus maintainers dedicated a part of their career to etcd and reviewed code, triaged bugs, and pushed the project forward over a substantial period of time. Thank you.
+These emeritus maintainers dedicated a part of their career to etcd and reviewed code, triaged bugs, and pushed the project forward over a substantial period of time. Thanks.
```

Comment:
This is being fixed under https://github.com/etcd-io/etcd/pull/10760 Since yours is a WIP, you may not need this change.  
The current build failure is due to your second commit is not in the expected format i.e. Expected commit title format '<package>{", "<package>}: <description>' 
Thanks!

Reply:
#10760 is merged. Please rebase to master so we don't double fix the same error. Sorry for the inconvenience.

Reply:
Thanks for update @spzala, @jingyih I have rebased and removed the readme file change.